packages/@rnw-scripts/create-github-releases/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
NIT: Nothing you can do, just sharing... :)
I wished yarn workspaces or lerna could deal with 'default' configs for packages... How many copies of just-task and tsconfig we have in our repo is getting a bit silly...

Reply:
We could potentially have a build script to populate them. I think we've done something similar before actually. I'm not sure it's worth the complexity yet, but could be a consideration if we see any maintenance headaches with these.